src/libConsensus/ConsensusLeader.h

```diff
@@ -62,7 +62,6 @@ class ConsensusLeader : public ConsensusCommon {
   std::condition_variable cv_scheduleSubsetConsensus;
   bool m_allCommitsReceived;
 
-  // TODO: the vectors should be replaced by more space efficient DS
```

Comment:
This was more of an issue back when we could not forecast the final size of a shard.  At 600 nodes (or even a few hundred more beyond), the use of vectors here should be ok, given that the size of the data and the lifetime of a consensus are both not very large.